docs/quickstart/run_example_quickstart.md

```diff
@@ -277,8 +277,8 @@ The number in front of each line tells you on which parallel instance of the pri
 was produced.
 
 This should get you started with writing your own Flink programs. You can check out our guides
-about [basic concepts]{{{ site.baseurl }}/apis/common/index.html} and the
-[DataStream API]{{{ site.baseurl }}/apis/streaming/index.html} if you want to learn more. Stick
+about [basic concepts]({{ site.baseurl }}/apis/common/index.html) and the
+[DataStream API]({{ site.baseurl }}/apis/streaming/index.html) if you want to learn more. Stick
```

Comment:
"if you want to learn more" -> "to learn more"

We can assume our users want to learn more :)